client/src/components/all-components/BrandedALLModal.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import { Modal } from "reactstrap";
 import PropTypes from "prop-types";
-import logo from "../../assets/images/logos/ALL_White.png";
+import logo from "../../assets/images/logos/ALL_White.svg";
 
-const BrandedALLModal = (props) => {
-  const { isOpen, toggle, direction, body } = props;
+const BrandedALLModal = (props, { children }) => {
+  const { isOpen, toggle, direction, body, width } = props;
 
   return direction === "row" && window.innerWidth >= 640 ? (
```

Comment:
window.innerWidth is used because if the screen is under 640px, it should only use a column modal because the row modal becomes too small for it to be functional on a small screen.

Reply:
can't this logic be handled using tailwind feature?

`column sm:row`?